courseraresearchexports/exports/utils.py

```diff
@@ -50,16 +50,17 @@ def download_export(export_job, dest):
     Download an export archive if ready given an export_job_id
     :param export_job:
     :param dest:
-    :return:
+    :return output_filename:
     """
-    if export_job['exportType'] == 'RESEARCH_WITH_EVENTING':
+    if export_job['exportType'] == EXPORT_TYPE_EVENTING:
         logging.error('Generate download links to download eventing exports')
-        raise ValueError('Require RESEARCH_WITH_SCHEMAS job')
+        raise ValueError('Require {} job'.format(EXPORT_TYPE_SCHEMAS))
```

Comment:
let's try and make more useful error messages - refer to using our app to download eventing exports somehow?